pkg/actors/actors.go

```diff
@@ -239,13 +241,20 @@ func (a *actorsRuntime) Init() error {
 			},
 			AfterTableUpdateFn: func() {
 				a.drainRebalancedActors()
-				a.actorsReminders.OnPlacementTablesUpdated(context.TODO())
+				a.actorsReminders.OnPlacementTablesUpdated(ctx)
```

Comment:
This cannot be `ctx` and must be a background context, because this callback is invoked periodically in background by the placement service (when placement tables are updated)

For now, maybe context.TODO()?


```suggestion
				a.actorsReminders.OnPlacementTablesUpdated(context.TODO())
```

Reply:
Oh, looks like the context is not just for init but for the entire lifecycle of actors? Please confirm if that's the case

Reply:
That's right- the context coming into `Init` will be the runner manager context which originates from the base signal context.